### ISSUE TITLE
Allow specifying alpha as a number from 0 to 255

### ISSUE THREAD

**Issue by Richienb** [sindresorhus/rgb-hex#9]
Currently, the alpha can only be specified as a decimal or percentage. However, it would be useful if it could be specified as a number from 0 to 255. Conversion back to percentages is: `alpha / 255`

```js
const rgbHex = require("rgb-hex")

rgbHex(255, 255, 255, 255)
//=> "ffffffff"
```

cc @sindresorhus

**Comment by sindresorhus** [comment]
What's the use-case? I don't see how that can be done with the suggested API as it's ambigious whether `rgbHex(255, 255, 255, 1)` means  `100%` alpha or `1/255%`.

**Comment by Richienb** [comment]
@sindresorhus A [Uint8ClampedArray](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8ClampedArray) encodes red, green, blue and alpha as integers from 0 to 255.

**Comment by sindresorhus** [comment]
Any suggestions on how the API should look like?

**Comment by Richienb** [comment]
@sindresorhus We could add an option that changes the behaviour of the alpha parameter.

**Comment by sindresorhus** [comment]
Yeah, I guess.

**Comment by sindresorhus** [comment]
I don't think this is worth adding. The use case is niche (mainly Uint8ClampedArray data), and any unambiguous API (like an options object) adds complexity that isn't justified for something easily handled in userland:

```js
rgbHex(255, 255, 255, alpha / 255)
```
